packages/react-query/tsup.config.ts

```diff
@@ -1,4 +1,4 @@
 import { options } from '@suspensive/tsup'
 import { defineConfig } from 'tsup'
 
-export default defineConfig(options)
```

Comment:
To prevent chunks from being created during the build, I specified only `index.ts` as the entry point. If chunks are created, they can unnecessarily accumulate during the file overwriting process, and unused chunks are not removed.